composer.json

```diff
@@ -24,16 +24,22 @@
         "source": "https://github.com/sulu/skeleton",
         "docs": "https://docs.sulu.io/"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:alexander-schranz/phpcr-migrations-bundle.git"
+        }
+    ],
     "require": {
         "php": "^8.0 || ^8.1",
         "ext-dom": "*",
         "ext-json": "*",
         "ext-simplexml": "*",
         "composer/package-versions-deprecated": "^1.8",
         "contao/imagine-svg": "^1.0",
-        "dantleech/phpcr-migrations-bundle": "^1.3",
+        "dantleech/phpcr-migrations-bundle": "dev-feature/test-phpcr-bundle-3-0 as 1.4.0",
         "doctrine/annotations": "^1.2 || ^2.0",
-        "doctrine/cache": "^1.0",
+        "doctrine/cache": "^1.0.1",
```

Comment:
we should also remove this requirement from Sulu this would mostly mean small BC Breaks or create a copy of ArrayCache of Doctrine.